package.json

```diff
@@ -34,12 +34,13 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "event-stream": "3.3.4",
     "gulp-concat": "2.6.1",
     "gulp-footer": "2.0.1",
```

Comment:
_gulp-footer_ [uses a `*` wildcard](https://github.com/tracker1/gulp-footer/blob/c2ccf6608223096dcf74c94c615620b8d87abd0c/package.json#L54) to just take any latest version, which is now 4.0.1 (not the nefarious/pulled 3.3.6). I would ideally like to wait for your PR [13](https://github.com/tracker1/gulp-footer/pull/13) to complete, then update the reference to _gulp-footer_ here, but I am not entirely opposed to proceeding without. Maybe we'll give it a few days to see what happens with that PR since there's no immediate urgency to close this PR since _event-stream_ was locked to 3.3.4. What do you think?

Reply:
Sounds sensible to me 👍

Reply:
Looks like we may get an updated _gulp-footer_ soon as @tracker1 might get to it this week.

Reply:
And we did. 2.0.2 dropped last night with event-stream removed, thanks to @Basa0's PR.